front/src/common/Map/Layers/extensions/SNCF/NeutralSections.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { LayerProps, Source } from 'react-map-gl/maplibre';
+
+import { RootState } from 'reducers';
+import { MAP_URL } from 'common/Map/const';
+
+import OrderedLayer from 'common/Map/Layers/OrderedLayer';
+import NeutralSectionSigns from 'common/Map/Layers/extensions/SNCF/NeutralSectionSigns';
+import type { Theme } from 'types';
+
+interface NeutralSectionProps {
```

Comment:
use a type instead of an interface
```suggestion
type NeutralSectionProps {
```